scriptable-transform-app-dependencies/pom.xml

```diff
@@ -1,52 +1,53 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <groupId>org.springframework.cloud.stream.app</groupId>
-    <artifactId>scriptable-transform-app-dependencies</artifactId>
-    <version>2.1.0.BUILD-SNAPSHOT</version>
-    <packaging>pom</packaging>
-    <name>scriptable-transform-app-dependencies</name>
-    <description>Spring Cloud Stream Scriptable Transform App Dependencies</description>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.springframework.cloud.stream.app</groupId>
+	<artifactId>scriptable-transform-app-dependencies</artifactId>
+	<version>2.1.0.BUILD-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<name>scriptable-transform-app-dependencies</name>
+	<description>Spring Cloud Stream Scriptable Transform App Dependencies</description>
 
-    <parent>
-	<artifactId>spring-cloud-dependencies-parent</artifactId>
-        <groupId>org.springframework.cloud</groupId>
-        <version>2.1.0.RC3</version>
-        <relativePath />
-    </parent>
+	<parent>
+		<artifactId>spring-cloud-dependencies-parent</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>2.1.0.BUILD-SNAPSHOT</version>
```

Comment:
Are those fixes going to work with spring-cloud-dependencies `2.1.0.RC3` as well? Or they depend on some recent change available only in 2.1.0.BUILD-SNAPSHOT? 
In other words, would this work with Einstein milestones?

Reply:
You are missing the fact, that the story here is not about SCSt, this is a root of all Spring Cloud projects.
And that's indeed what we need from there, because this one has not been released yet: https://github.com/spring-projects/spring-boot/issues/15468

Reply:
* The `groovy-dateutil`, `groovy-jaxb` and  `groovy-datetime` dependencies are only available in the BOOT's master bom but **not** present in the [BOOT 2.1.1.RELEASE](https://github.com/spring-projects/spring-boot/blob/v2.1.1.RELEASE/spring-boot-project/spring-boot-dependencies/pom.xml).
* The [spring-cloud-build master (e.g. 2.1.0.BUILD-SNAPSHOT) BOM](https://github.com/spring-cloud/spring-cloud-build/blob/master/spring-cloud-build-dependencies/pom.xml) is includes **not** the master but the [BOOT 2.1.1.RELEASE](https://github.com/spring-projects/spring-boot/blob/v2.1.1.RELEASE/spring-boot-project/spring-boot-dependencies/pom.xml) - meaning it is missing the `groovy-dateutil`, `groovy-jaxb` and  `groovy-datetime` dependencies 
* In turn that means that those dependencies are missing from all App Starters's BOM including `scriptable-transform`. 

As a result the `scriptable-transform` won't compile unless we explicitly add the above dependencies to the `scriptable-transform-app-dependencies`. At least until the spring-cloud-build is upgraded to a post 2.1.1 boot milestone/release.